pkg/generated/clientset/versioned/clientset.go

```diff
@@ -9,57 +9,25 @@ import (
 	discovery "k8s.io/client-go/discovery"
 	rest "k8s.io/client-go/rest"
 	flowcontrol "k8s.io/client-go/util/flowcontrol"
-	configmanagementv1 "kpt.dev/configsync/clientgen/apis/typed/configmanagement/v1"
-	configsyncv1alpha1 "kpt.dev/configsync/clientgen/apis/typed/configsync/v1alpha1"
-	configsyncv1beta1 "kpt.dev/configsync/clientgen/apis/typed/configsync/v1beta1"
-	hubv1 "kpt.dev/configsync/clientgen/apis/typed/hub/v1"
-	kptv1alpha1 "kpt.dev/configsync/clientgen/apis/typed/kpt.dev/v1alpha1"
+	configmanagementv1 "kpt.dev/configsync/pkg/generated/clientset/versioned/typed/configmanagement/v1"
 )
 
 type Interface interface {
 	Discovery() discovery.DiscoveryInterface
 	ConfigmanagementV1() configmanagementv1.ConfigmanagementV1Interface
-	ConfigsyncV1alpha1() configsyncv1alpha1.ConfigsyncV1alpha1Interface
-	ConfigsyncV1beta1() configsyncv1beta1.ConfigsyncV1beta1Interface
-	HubV1() hubv1.HubV1Interface
```

Comment:
Do you know why are there APIs deleted?

Reply:
Good catch, looks like it's because those packages were missing the `+genclient` comment